libraries/resource_firewall_rule.rb

```diff
@@ -22,6 +22,9 @@ class Resource::FirewallRule < Resource
     attribute(:redirect_port, :kind_of => Integer)
     attribute(:description, :kind_of => String, :name_attribute => true)
 
+    # Indicate whether this rule needs to be persistent on the system
+    attribute(:persist, :kind_of => Integer, :equal_to => [0, 1], :default => 0)
```

Comment:
How will this be handled on the other providers of firewall_rule? It would be good to handle this similarly across all of them. In fact, some of them only persist at the firewall level, not at the individual rule level.